docs/topics/build-tools-api.md

```diff
@@ -0,0 +1,104 @@
+[//]: # (title: Build tools API)
+
+<primary-label ref="experimental-general"/>
+
+<!-- Add diagram -->
+
+Kotlin 2.2.0 introduces an experimental Build tools API (BTA) that simplifies how build systems integrate with the 
+Kotlin compiler.
+
+Traditionally, adding full Kotlin support to a build system (like incremental compilation, Kotlin compiler plugins, 
+daemons, and Kotlin Multiplatform) required significant effort. The BTA aims to reduce this complexity by providing
+a unified API between build systems and the Kotlin compiler ecosystem.
+
+The BTA defines a single entry point that build systems can implement. This removes the need to deeply integrate with internal compiler details.
+
+Currently, the BTA supports Kotlin/JVM **only**.
+
+> The BTA itself is not yet publicly available for direct use in your own build tool integrations.
+> If you're interested in the proposal or want to share feedback, see the [KEEP](https://github.com/Kotlin/KEEP/issues/421).
+> 
+{style="warning"}
+
+## Integration with Gradle
+
+The Kotlin Gradle plugin (KGP) already supports the BTA.
```

Comment:
```suggestion
The Kotlin Gradle plugin (KGP) already supports the BTA. The support is an [experimental feature](components-stability.md#stability-levels-explained) that may be changed at any time. We would appreciate your feedback in [YouTrack](https://youtrack.jetbrains.com/issue/KT-56574)
```

Reply:
Added experimental label also to this section and a separate section for sharing feedback on the experience with the KGP.